activestorage/app/jobs/active_storage/preview_image_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ActiveStorage::PreviewImageJob < ActiveStorage::BaseJob
+  queue_as { ActiveStorage.queues[:preview_image] }
```

Comment:
Does this need to be documented? ref https://guides.rubyonrails.org/configuring.html#config-active-storage-queues-analysis

Reply:
Yes!

Reply:
https://github.com/rails/rails/pull/51113